### ISSUE TITLE
Add client-side validation helpers

### ISSUE THREAD

**Issue by davidmurdoch** [stripe/stripe-node#155]
`card.validateCardNumber`, `card.validateExpiry`, `card.validateCVC`, `card.cardType` and `bankAccount.validateRoutingNumber` `bankAccount.validateAccountNumber` are in stripe.js but not in stripe-node.

**Comment by APTy** [comment]
I think you can actually get access to those helper functions by adding the [jquery.payment library](https://github.com/stripe/jquery.payment) to your project (though some of the function names are slightly different). Is that what you're looking for?

**Comment by SlexAxton** [comment]
(Hey @davidmurdoch - fancy seeing you here) For the most part, this is intentional. You shouldn't have card numbers in node to do validation on, or else you've got major PCI problems. And you shouldn't be browserifying stripe-node into the client, because it requires a private key to communicate. So both of the reasons I could see for doing this seem scary. You got another?

**Comment by davidmurdoch** [comment]
*waves*

This is actually for an Android app, and the Debit/CC numbers would only ever be in the server's RAM (or possibly OS swap-space), which I just think/thought was allowed under PCI/DSS compliance.

Reading up on Stripe+Android I think I've found my answer: I'd need to validate and tokenize the numbers directly within the Android app, then just send the token to my app when I want to do stuff with money. So it does make sense that these helpers functions are not in stripe-node.

Unrelated:
At first glance, I didn't see anywhere to create an app-specific secret key with limited privileges, e.g., an app that allows tokenizing credits cards, but not charges, transfers, etc. Needing to storing an end-all-be-all PK in the app makes me a little uncomfortable.

Also, I just realized that I can't transfer money to my Stripe account *from* my bank, so Stripe might not work for me anyway, as my profit margins are already pretty slim; I'm looking into if buying Bitcoin then transferring it to Stripe will make sense.

Thanks!
